l/lego/lego_ubi8.3.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package        : Lego
+# Version        : 3.5.0 - latest
+# Source repo    : https://github.com/go-acme/lego
+# Tested on      : Ubi 8.3 + Go latest version (v1.7.1)
+# Script License : Apache 2.0
+# Maintainer     : Eshant Gupta <eshant.gupta@ibm.com>; Priya Seth <sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# Usage: ./lego_ubi_8.3.sh
+#                  OR
+#        ./lego_ubi_8.3.sh v3.5.0
+# ----------------------------------------------------------------------------
+
+source ~/.profile
+set_env=false
+
+# Get sudo access
+[ "$UID" -eq 0 ] || exec sudo bash "$0" "$@"
+
+function usage() {
+        echo "$1 accepts 0 or maximum 1 argument"
+        exit 1
+}
+
+if [ "$#" -gt 1 ]
+then
+        usage "$(basename $0)"
+else
+        [ -n "$1" ] && export TAGS="$1" || export TAGS=""
+fi
+
+# check installed Golang
+[ -n $(which go) 2>&1 > /dev/null ] && [[ $(go version) > 1.6 ]] && set_env=true
+
+# setup Golang
+if ! $set_env
+then
+
+    cd "$HOME/$USER"
+    mkdir golang go && sudo yum install git wget curl make gcc -y
+    [ $? != 0 ] && echo "Installation Failed" && exit 1
+
+    echo "Finding latest version of Go for PPC64LE..."
+    latest=$(curl https://golang.org/VERSION?m=text)
+
+    echo "Downloading latest Go for PPC64LE: ${latest}"
+    $(wget https://dl.google.com/go/${latest}.linux-ppc64le.tar.gz)
+    tar -C golang -xzf "${latest}".linux-ppc64le.tar.gz
+
+    echo "Create the skeleton for your local users go directory"
+    mkdir -p go/{bin,pkg,src}
+
+    echo "Setting up GOPATH"
+    echo "export GOPATH=$(pwd)/go" >> .profile && source .profile
+
+    echo "Setting PATH to include golang binaries"
+    echo "export PATH='$PATH':$(pwd)/golang/go/bin:$GOPATH/bin" >> .profile && source .profile
+
+fi
+
+# install lego package
+GO111MODULE="off" go get github.com/go-acme/lego
+cd $GOPATH/src/github.com/go-acme/lego
+go get -tags $TAGS -t ./...
+make test && make build
+[ $? != 0 ] && exit 1
```

Comment:
`set -eu` can be used at the beginning of the script to get rid of these command status checks.